packages/server/modules/automations/helpers/inputTypes.ts

```diff
@@ -0,0 +1,52 @@
+import { AutomationRunStatus } from '@/modules/core/graph/generated/graphql'
+import { z } from 'zod'
+
+const CurrentObjectResultsVersion = '23.09.2023' as const
```

Comment:
Moved all of these to a separate helper so that the module is easier to understand. Essentially all of these only relate to input types and input validation, when we output things we use either DB or GQL types.

Reply:
This was (20)23.09 as in ubuntu calver

Reply:
Never would've guessed that haha, I'd just use semantic versioning here honestly

Reply:
+ YY.MM would only allow for a single version per month